tests/test_app.py

```diff
@@ -436,3 +439,18 @@ def test_warning_config_options_on_compile_command(snippetcompiler, tmpdir):
     assert "Compile done" in all_output
     assert f"Config directory {non_existing_config_dir} doesn't exist" not in all_output
     assert f"Config file {non_existing_config_file} doesn't exist" not in all_output
+
+
+@pytest.mark.parametrize(
+    "with_tty, regexes_required_lines, regexes_forbidden_lines",
+    [(False, [r"Current Inmanta version:"], []), (True, [r"Current Inmanta version:"], [])],
+)
+@pytest.mark.timeout(20)
+def test_version_argument_is_set(tmpdir, with_tty, regexes_required_lines, regexes_forbidden_lines):
```

Comment:
This test case only tests if the output starts with "Current Inmanta version:". We should verify the version number as well. Otherwise we are never sure if the "--version" command outputs the actual version number. 

Maybe it's also a good idea to test the negative scenario. I.e. test if version is not shown when the --version option is not present.

Reply:
@arnaudsjs where is the testcase going to get the number?

Reply:
=> get_compiler_version()

Reply:
But the source implementation does the same, they could both be wrong and the test would pass